tests/cephfs/ceph-83573462.py

```diff
@@ -0,0 +1,195 @@
+            long_running=True,
+        )
+        for i in range(10):
+            dir_name_generate = "".join(
+                random.choice(string.ascii_lowercase + string.digits)
+                for _ in list(range(5))
+            )
+            client1.exec_command(
+                sudo=True,
+                cmd="mkdir %s%s" % (kernel_mounting_dir, "dir_" + dir_name_generate),
+            )
+        fuse_dir_generate = "".join(
+            random.choice(string.ascii_lowercase + string.digits)
+            for _ in list(range(5))
+        )
+        fuse_mounting_dir = f"/mnt/cephfs_fuse{fuse_dir_generate}/"
+        client2.exec_command(sudo=True, cmd="dnf install ceph-fuse")
+        fs_util.auth_list([client2])
+        fs_util.fuse_mount([client2], fuse_mounting_dir)
+        for i in range(10):
+            dir_name_generate = "".join(
+                random.choice(string.ascii_lowercase + string.digits)
+                for _ in list(range(5))
+            )
+            client2.exec_command(
+                sudo=True,
+                cmd="mkdir %s%s" % (fuse_mounting_dir, "dir_" + dir_name_generate),
+            )
+        c1_out, c1_result = client1.exec_command(
+            sudo=True, cmd="ceph fs get cephfs -f json"
+        )
+        decoded_out = json.loads(c1_out.read().decode())
+        number_of_up_temp = decoded_out["mdsmap"]["up"]
+        number_of_up = len(number_of_up_temp)
+        number_of_mds_max = decoded_out["mdsmap"]["max_mds"]
+        c1_out2, result2 = client1.exec_command(sudo=True, cmd="ceph -s -f json")
+        decoded_out2 = json.loads(c1_out2.read().decode())
+        number_of_standby = decoded_out2["fsmap"]["up:standby"]
+        log.info(number_of_standby)
+        counts = number_of_standby
+        for i in range(counts):
+            number_of_mds_max = number_of_mds_max + 1
+            client1.exec_command(
+                sudo=True, cmd=f"ceph fs set cephfs max_mds {str(number_of_mds_max)}"
+            )
+            number_of_standby = number_of_standby - 1
+            number_of_up = number_of_up + 1
+            time.sleep(50)
+            kernel_output, kernel_result = client1.exec_command(
+                sudo=True, cmd="ceph fs get cephfs -f json"
+            )
+            kernel_decoded = json.loads(kernel_output.read().decode())
+            current_max_mds = kernel_decoded["mdsmap"]["max_mds"]
+            kernel_output2, kernel_result2 = client1.exec_command(
+                sudo=True, cmd="ceph -s -f json"
+            )
+            kernel_decoded2 = json.loads(kernel_output2.read().decode())
+            current_standby = kernel_decoded2["fsmap"]["up:standby"]
+            if current_max_mds != number_of_mds_max:
+                return 1
+            if number_of_up != number_of_mds_max:
+                return 1
+            if number_of_standby != current_standby:
+                return 1
```

Comment:
Can we have a common function for validating max_mds i.e., from 86 to 109. As we are repeating same in later steps with minimal changes. with **number_of_mds_max** as argument to the function

Reply:
How about **def number_of_mds_max( client, mds_max)**?
I will make it simpler and will ask for review again.